lib/src/chart/base/axis_chart/axis_chart_data.dart

```diff
@@ -1129,14 +1129,16 @@ class VerticalLine extends FlLine with EquatableMixin {
 /// Draws a title on the [HorizontalLine]
 class HorizontalLineLabel extends FlLineLabel with EquatableMixin {
   /// Draws a title on the [HorizontalLine], align it with [alignment] over the line,
-  /// applies [padding] for spaces, and applies [style for changing color,
+  /// applies [padding] for label padding, applies [verticalOffset] and [horizontalOffset] for label space and applies [style] for changing color,
   /// size, ... of the text.
   /// Drawing text will retrieve through [labelResolver],
   /// you can override it with your custom data.
   /// [show] determines showing label or not.
   /// [direction] determines if the direction of the text should be horizontal or vertical.
   HorizontalLineLabel({
     super.padding = const EdgeInsets.all(6),
```

Comment:
I think it doesn't make sense to have padding in HorizontalLineLabel and VerticalLineLabel anymore. 
Because we can now use horizontalOffset and verticalOffset instead.
Correct?

Reply:
I think the whole point for the PR was to make padding for the Horizontal/VerticalLineLabel, see [Issue 824](https://github.com/imaNNeo/fl_chart/issues/824). It seems that the background color from the text style is broken atm, I'll fix that.

Reply:
Padding example:

![img](https://github.com/user-attachments/assets/d5dd2569-acfd-45ac-8193-f4f3a65af0d1)